frontend/src/components/study-card-images/card-images.tsx

```diff
@@ -1,65 +1,51 @@
-export const BaseUrl = 'https://kinetic-app-assets.s3.amazonaws.com/assets/card-images'
-
 export type Category =
-    'Learning' |
-    'Memory' |
-    'Personality' |
-    'School & Career' |
-    'Other'
+    'Biology Learning' |
+    'Curated Studies' |
+    'Future Careers' |
+    'Growth & Resilience' |
+    'Interpersonal Skills' |
+    'Learning Persistence' |
+    'Mental Agility' |
+    'Personal Finance' |
+    'Productivity' |
+    'STEM Career' |
+    'Study Strategies'
 
 export const imageCategories: Category[] = [
-    'Learning',
-    'Memory',
-    'Personality',
-    'School & Career',
-    'Other',
+    'Biology Learning',
+    'Curated Studies',
+    'Future Careers',
+    'Growth & Resilience',
+    'Interpersonal Skills',
+    'Learning Persistence',
+    'Mental Agility',
+    'Personal Finance',
+    'Productivity',
+    'STEM Career',
+    'Study Strategies',
 ]
 
 interface CardImage {
     imageId: string,
-    category: Category[],
+    category: Category,
     altText: string
 }
 
-// TODO These can be removed once we update everything on prod
-const legacyImageMap: Record<string, string> = {
-    'AbilityBeliefs': 'Schoolfuturecareer_19',
-    'AnxiousPersonNew': 'Personality_1',
-    'AreYouSuperReader': 'MemoryLearning_2',
-    'Biology1': 'Learning_6',
-    'Biology2': 'Learning_3',
-    'Biology3': 'Other_1',
-    'ControlOverYourLife': 'Personality_3',
-    'Demographic': 'Demographic',
-    'DoYouFitIn': 'Personality_4',
-    'DoYouStickWithGoal': 'Schoolfuturecareer_16',
-    'FinancialLiteracy': 'Other_9',
-    'HowDoYouStudyOnline': 'Learning_8',
-    'HowImpulsiveAreYou': 'Personality_7',
-    'HowManyLettersYouRemember': 'MemoryLearning_1',
-    'HowMuchYouRemeberFromScience': 'Memory_10',
-    'NewPersonalityTraits': 'Personality_6',
-    'PatternRecognition': 'Memory_11',
-    'PotentialFutureCareer': 'Schoolfuturecareer_18',
-    'Psychology': 'Learning_9',
-    'RememberNumbersWhenReading': 'MemoryLearning_1',
-    'StemInterest': 'Schoolfuturecareer_21',
-    'UncoverAchievementLearning': 'Schoolfuturecareer_22',
-}
-
 export const getImageUrl = (imageId: string | undefined) => {
     if (!imageId) {
-        return `${BaseUrl}/placeholder.svg`
+        return `https://kinetic-app-assets.s3.amazonaws.com/assets/card-images/placeholder.svg`
```

Comment:
Note for future - we can just add the placeholder.svg in the public directory and references that , no need to handle that right now though